languages/go/exercises/concept/maps/.docs/after.md

```diff
@@ -0,0 +1,58 @@
+Further Reading:
+
+- [godoc-race-detector]
+- [goblog-race-detector]
+- [goblog-iteration-order]
```

Comment:
I would suggest removing these additional reading links, as you also mention them at the end of the document and I think that makes more sense then starting with them :)

```suggestion
Further Reading:

- [godoc-race-detector]
- [goblog-race-detector]
- [goblog-iteration-order]
```

Reply:
Hmm, my intention is to give the reader some quick links to refer to, what do you think?

Reply:
I would still prefer them to be at the end of the file. My reason for this is that I would like this document to make sense when reading from top to bottom. This is like a deep-dive before explaining the basics. The document also isn't that big that the links will go unnoticed when at the bottom of the file.

Reply:
Alright, I fixed it in 46fb946

Reply:
Thanks!